spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -288,6 +288,12 @@ sealed class SpdxSingleLicenseExpression : SpdxExpression() {
      * Return the license exception identifier if this is a [SpdxLicenseWithExceptionExpression] or null otherwise.
      */
     abstract fun exception(): String?
+
+    /**
+     * Return the URL to https://spdx.org for the licence if this is [SpdxLicenseIdExpression] or
```

Comment:
Similarly here, the fact that the URL (currently) points to https://spdx.org is an implementation detail that we should not mention as part of the docs.

Reply:
True. Removed.